pyproject.toml

```diff
@@ -77,32 +77,39 @@ keywords = [
 ]
 
 [project.optional-dependencies]
+# LINT.IfChange
 examples-common = [
   "gunicorn>=20.1.0",
+  "tensorflow>=2.16.0",
+  "transformers>=4.27.1",
 ]
+# LINT.ThenChange(./requirements_examples_common.txt)
 # LINT.IfChange
-examples = [
+examples-descai = [
```

Comment:
what is descai?

Reply:
Short for Descriminative AI (aka clasification and regression).